source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.tsx

```diff
@@ -135,12 +136,10 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
     const acquisitionProperties = await acquisitionPropertiesTask;
     const acquisitionChecklist = await acquisitionChecklistTask;
 
-    if (retrieved) {
-      retrieved.fileProperties = acquisitionProperties;
-      retrieved.fileChecklistItems = acquisitionChecklist ?? [];
-      setFile({ ...retrieved, fileType: FileTypes.Acquisition });
-      setStaleFile(false);
-    }
+    retrieved.fileProperties = acquisitionProperties ?? null;
```

Comment:
my instinct here is ?? []. why ?? null?

Reply:
right now we assume that lists could be null. We could change the backend serializer to return empty lists instead